### PR TITLE
mtx lora. support dlms based toJson method

### PR DESCRIPTION
**Pull request by Pnom**

### CODE REVIEW

File: src/mtxLora/IDlmsJsonOptions.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import {TJsonOptions, defaultJsonOptions} from './Command.js';
```
Comment (DarkPark) [inline src/mtxLora/IDlmsJsonOptions.ts:1]: src/mtxLora/dlms.ts

?

or put it in CommandBinaryBuffer

File: src/mtxLora/commands/uplink/GetDayEnergies.ts [hunk 2]
```diff
@@ -107,6 +141,19 @@ class GetDayEnergies extends Command {
             ...buffer.getBytesToOffset()
         ]);
     }
+
+    toJson ( {dlms}: IDlmsJsonOptions = defaultDlmsJsonOptions ) {
```
Comment (DarkPark) [inline src/mtxLora/commands/uplink/GetDayEnergies.ts:145]: defaultJsonOptions are not used here?

File: src/mtxLora/commands/uplink/GetHalfhoursEnergies.ts [hunk 3]
```diff
@@ -118,6 +150,26 @@ class GetHalfhoursEnergies extends Command {
             ...buffer.getBytesToOffset()
         ]);
     }
+
+    toJson ( {dlms}: IDlmsJsonOptions = defaultDlmsJsonOptions ) {
+        const {parameters} = this;
+        const {
```
Comment (DarkPark) [inline src/mtxLora/commands/uplink/GetHalfhoursEnergies.ts:156]: it looks very sparse but gives not very much

File: tests/mtxLora/get.day.energies.dlms.test.ts [hunk 4]
```diff
@@ -0,0 +1,415 @@
+/* eslint-disable */
```
Comment (DarkPark) [inline tests/mtxLora/get.day.energies.dlms.test.ts:1]: can we enable it?

File: src/mtxLora/commands/uplink/GetDayEnergies.ts (hunk 2 above)
Comment (Pnom) [reply src/mtxLora/commands/uplink/GetDayEnergies.ts:144]: it's default value

File: src/mtxLora/commands/uplink/GetHalfhoursEnergies.ts (hunk 3 above)
Comment (Pnom) [reply src/mtxLora/commands/uplink/GetHalfhoursEnergies.ts]: fixed

File: tests/mtxLora/get.day.energies.dlms.test.ts (hunk 4 above)
Comment (Pnom) [reply tests/mtxLora/get.day.energies.dlms.test.ts]: fixed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**